frontend/ui/data/onetime_migration.lua

```diff
@@ -566,5 +567,24 @@ if last_migration_date < 20230703 then
     end
 end
 
+-- 20230707, OPDS, no more special calibre catalog
+if last_migration_date < 20230707 then
+    logger.info("Performing one-time migration for 20230707")
+
+    local calibre_opds = G_reader_settings:readSetting("calibre_opds")
+    if next(calibre_opds) then
```

Comment:
If that plugin happened to have been disabled (at install time, for some reason, or what do we know why :), you may get calibre_opds = nil, so next() will crash. Better check for nil.